vllm/sequence.py

```diff
@@ -155,18 +169,20 @@ def output_token_ids(self) -> Tuple[int, ...]:
         return tuple(self._output_token_ids)
 
     @output_token_ids.setter
-    def output_token_ids(self, new_output_token_ids) -> None:
-        self._output_token_ids = array('l', new_output_token_ids)
+    def output_token_ids(self, new_output_token_ids: List[int]) -> None:
+        self._output_token_ids = array('I', new_output_token_ids)
```

Comment:
Note: We have to be careful about the array type if we will end up sending it to be a PyTorch tensor. `I` is 2 bytes while `torch.long` (aka `torch.int64`) is 4 bytes.

Reply:
It is fine now because you can only access array when you use *array API. I added warning to the API about this.